Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props => {
+    return (
+        <div className="header-title-container">
+            <h2 className="header-title">
+                <a href={props.user.profile}>{props.user.name}</a>
+            </h2>
+            <a className="user-handle" href={props.user.profile}>
+                {props.user.handle}
+            </a>
+            <p className="post-date">26 jan</p>
```

Comment:
This component looks like a pretty good candidate for object destructuring. For example:
```javascript
const HeaderTitle = ({ user }) => {
  return (
    <div className="header-title-container">
      <h2 className="header-title">
        <a href={user.profile}>{user.name}</a>
      </h2>
      <a className="user-handle" href={user.profile}>
         {user.handle}
      </a>
      <p className="post-date">26 jan</p>
    </div>
  );
};
```
or going even further:
```javascript
const HeaderTitle = ({ user }) => {
  const { profile, name, handle } = user;
  return (
    <div className="header-title-container">
      <h2 className="header-title">
        <a href={profile}>{name}</a>
      </h2>
      <a className="user-handle" href={profile}>
         {handle}
      </a>
      <p className="post-date">26 jan</p>
    </div>
  );
};
```